Reply/app/src/main/res/values/color.xml

```diff
@@ -14,34 +14,64 @@
   -->
 <resources>
 
-    <!--Color palette-->
-    <color name="reply_white_50">#ffffff</color>
+    <!--Light Colours-->
```

Comment:
nit: Colour -> Color to match the code (`<color`, `colorPrimary`, etc)

Reply:
Oops, Canadian Spelling.